src/bodyScrollLock.js

```diff
@@ -1,10 +1,4 @@
-// @flow
-import UAParser from 'ua-parser-js';
-// Adopted and modified solution from Bohdan Didukh (2017)
-// https://stackoverflow.com/questions/41594997/ios-10-safari-prevent-scrolling-behind-a-fixed-overlay-and-maintain-scroll-posi
-
-const parser = new UAParser();
-const isIosDevice = typeof window !== 'undefined' && parser.getOS().name === 'iOS';
+const isIosDevice = !!navigator.platform && /iPad|iPhone|iPod/.test(navigator.platform);
```

Comment:
can you add `typeof navigator !== 'undefined'` in case this code gets loaded on the server side (ie. where navigator/window is undefined)?

So: `const isIosDevice = typeof navigator !== 'undefined' && !!navigator.platform && /iPad|iPhone|iPod/.test(navigator.platform);`